src/GameSync.Api.Integration/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Diagnostics.CodeAnalysis;
```

Comment:
Let's have one global `GlobalSuppressions.cs` at solution level. You can do it this way https://learn.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2010/9f4t9t92(v=vs.100)?redirectedfrom=MSDN

Reply:
Done. Thanks for this.